addons/html_builder/static/tests/helpers.js

```diff
@@ -305,10 +305,13 @@ export async function setupHTMLBuilder(
     comp.state.isEditing = true;
     await prom;
     await animationFrame();
+    const contentEl = [...comp.iframeRef.el.contentDocument.body.firstChild.childNodes].find(
```

Comment:
This seems a bit weird... `contentEl` used to be the first random P then? or the first editable content? How. comes it finds a selection placeholder?

Reply:
I just ran the `html_builder` tests without this change and everything passed so I think this is a relic from before adding all the exceptions in terms of where not to insert placeholders.

From a random test, I see that `comp.iframeRef.el.contentDocument.body.firstChild` is the `#wrapwrap` element so its first child is usually a snippet: 
<img width="1741" height="103" alt="image" src="https://github.com/user-attachments/assets/2263da02-b377-4178-a6b1-e1172914b884" />

I don't know why I used to have placeholders directly within the `#wrapwrap` given it's `contenteditable=false` but maybe there are tests where it's directly editable.